### PR TITLE
Fix variable's 'add to watch' feature.

### PR DESCRIPTION
**Pull request by rkeithhill**
Fix #838 in vscode-powershell

### CODE REVIEW

File: src/PowerShellEditorServices/Debugging/DebugService.cs [hunk 1]
```diff
@@ -603,7 +603,7 @@ public VariableDetailsBase GetVariableFromExpression(string variableExpression,
             // of command executions into string output.  However, if null is returned
             // then return null so that no output gets displayed.
             string outputString =
-                results != null ?
+                results != null && results.Count() > 0 ?
```
Comment (daviwil) [inline src/PowerShellEditorServices/Debugging/DebugService.cs]: Possibly more efficient to use results.Any() here?

Comment (rkeithhill) [inline src/PowerShellEditorServices/Debugging/DebugService.cs]: Um ... yeah, good call!  Much quicker than to count a 10000 element sequence.

**Comment by rkeithhill** [comment]
Before your merge a PR do you assign it a milestone or do you do that just for the related issue?

**Comment by daviwil** [comment]
I've been assigning milestones for both issues and PRs lately so I can keep track of what work got done.  In this case since we don't have a new version number milestone yet we can just assign it to the June milestone.  Once I'm ready to ship an update I scan through the closed items for the current month milestone and assign them to the specific version milestone.

I should write up something about the milestone model since it's gone through several variations before I settled on this ;)

**Pull request merged**